packages/devops/downloadenv.Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Fetch environment from LastPass and write to /env/<package>/.env
```

Comment:
packages/devops/docker if its not too annoying with layer invalidation

Reply:
Done in 697d87be786a16dfacaa14e14d27f3510a187940
Added packages/devops/docker to .dockergnore so invalidation won't be a problem